code/modules/clothing/head/misc.dm

```diff
@@ -280,5 +280,5 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
the joke should be the crossed out text alone, not crossed out text and shit grammar because if you don't fix it now someone WILL in the future.